File: python/ex19.py
velo= float(input("Digite a velocidade inicial V0:"))
acelera = float(input("entre com o valor de aceleracao em m/s:"))
tempo= float(input("Digite o tempo de percurso:"))

v=((velo+acelera*tempo)*3,6)

if (v<=40):
    print("veiculo muito lento")

elif(40<v<60):
    print("velocidade permitida")
elif(60<v<80):
    print("velocidade de cruzeiro")  
elif(80<v<120):
    print("veiculo rapido")    
else:
    print("veiculo muito rapido")          

File: ex19.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float  acelera, veloini, tper, velofinal;
    printf("digite a aceleracao do inicial:");
    scanf("%f", &acelera);

    printf("digite a velocidade inicial:");
    scanf("%f", &veloini);

    printf("digite o tempo do percurso:");
    scanf("%f", &tper);
  
    velofinal=veloini+acelera*tper;

    if (velofinal<=40){
     printf("veiculo muito lento \n %.f", velofinal);
    }
    else if (40<=velofinal<=60){
    printf("velocidade permitida \n %.f", velofinal) ;
    }
    else if (60<velofinal<=80){
    printf("velocidade de cruzeiro \n%.f", velofinal) ;    
    }
    else if (80<velofinal<=120){
    printf("veiculo rapido \n%.f",velofinal);   
    }
    else {
    printf("veiculo mjuito rapido \n %.f", velofinal);   
    }

   


    return 0;
}
